src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,78 @@
+import{ useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [birthday, setBirthday] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password,
+      Email: email,
+      Birthday: birthday
```

Comment:
@cinchronicity ,
I noticed that your backend API expects the keys to be lower case as seen here: https://github.com/cinchronicity/FlixHive-client/blob/9baf314bde3ab10e1947930db46f47bff7edd236/src/components/signup-view/signup-view.jsx#L12-L17 

Kind update these as below:-
```suggestion
      username: username,
      password: password,
      email: email,
      birthday: birthday
```